docs/conf.py

```diff
@@ -101,8 +104,7 @@
 
 # HTML theme
 html_theme = 'furo'
-html_logo = "images/logo.png"
-html_favicon = "images/favicon.ico"
```

Comment:
Maybe keep it pointing at SVG as a fallback?

Reply:
Once we agree in a good file structure, I can test this.
I suppose it will work fine, although it might obfuscate the setup for multiple sizes in some browser.